<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>motorpage</class>
 <widget class="QMainWindow" name="motorpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>电机参数可视化</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>电机参数可视化</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_61" stretch="1">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_45">
            <item>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="2" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_51"/>
              </item>
              <item row="0" column="0">
               <widget class="QPushButton" name="pushButton_5">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Show</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="Line" name="line_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="btn_test1">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_test2">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_testReadAll">
              <property name="text">
               <string>ReadAll</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_8" columnminimumwidth="3,1,3,1,1">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_46">
            <item>
             <widget class="QCustomPlot" name="qcustomplot1" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLineEdit" name="le_plot1">
                <property name="text">
                 <string>3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_plot1">
                <property name="text">
                 <string>Show</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_47">
            <item>
             <widget class="QCustomPlot" name="qcustomplot2" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_52">
              <item>
               <widget class="QLineEdit" name="le_plot2">
                <property name="text">
                 <string>3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_plot2">
                <property name="text">
                 <string>Show</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <layout class="QVBoxLayout" name="verticalLayout_48">
            <item>
             <widget class="QCustomPlot" name="qcustomplot3" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_53">
              <item>
               <widget class="QLineEdit" name="le_plot3">
                <property name="text">
                 <string>3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_plot3">
                <property name="text">
                 <string>Show</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="3">
           <layout class="QVBoxLayout" name="verticalLayout_49">
            <item>
             <widget class="QCustomPlot" name="qcustomplot4" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_54">
              <item>
               <widget class="QLineEdit" name="le_plot4">
                <property name="text">
                 <string>3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_plot4">
                <property name="text">
                 <string>Show</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="4">
           <layout class="QVBoxLayout" name="verticalLayout_50">
            <item>
             <widget class="QCustomPlot" name="qcustomplot5" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_55">
              <item>
               <widget class="QLineEdit" name="le_plot5">
                <property name="text">
                 <string>3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_plot5">
                <property name="text">
                 <string>Show</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_51">
            <item>
             <widget class="QCustomPlot" name="qcustomplot6" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_56">
              <item>
               <widget class="QLineEdit" name="le_plot6">
                <property name="text">
                 <string>3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_plot6">
                <property name="text">
                 <string>Show</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_52">
            <item>
             <widget class="QCustomPlot" name="qcustomplot7" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_57">
              <item>
               <widget class="QLineEdit" name="le_plot7">
                <property name="text">
                 <string>3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_plot7">
                <property name="text">
                 <string>Show</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="2">
           <layout class="QVBoxLayout" name="verticalLayout_53">
            <item>
             <widget class="QCustomPlot" name="qcustomplot8" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_58">
              <item>
               <widget class="QLineEdit" name="le_plot8">
                <property name="text">
                 <string>3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_plot8">
                <property name="text">
                 <string>Show</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="3">
           <layout class="QVBoxLayout" name="verticalLayout_54">
            <item>
             <widget class="QCustomPlot" name="qcustomplot9" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_59">
              <item>
               <widget class="QLineEdit" name="le_plot9">
                <property name="text">
                 <string>3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_plot9">
                <property name="text">
                 <string>Show</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="4">
           <layout class="QVBoxLayout" name="verticalLayout_55">
            <item>
             <widget class="QCustomPlot" name="qcustomplot10" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_60">
              <item>
               <widget class="QLineEdit" name="le_plot10">
                <property name="text">
                 <string>3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_plot10">
                <property name="text">
                 <string>Show</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>电机参数数据库</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Num:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Motor:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_Motor">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_Num">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QTableWidget" name="tb_motordata"/>
        </item>
        <item row="0" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Rage:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_rage_start">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_rage_end">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>100</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_showDB">
            <property name="text">
             <string>showDB</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_exportData">
            <property name="text">
             <string>export</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Round:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_round">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_deleteData">
            <property name="text">
             <string>dltData</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_nuke">
            <property name="text">
             <string>Nuke</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Num:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_totalDataNum">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>inc/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
